### ISSUE TITLE
Running the Dashboard on a remote machine

### ISSUE THREAD

**Issue by dmadeka** [NervanaSystems/coach#224]
Im trying to run the Coach Dashboard for the training happening in a remote machine. Unfortunately, I get the error 

```bash
ubuntu@ip-172-31-26-79:~$ ~/anaconda3/envs/tensorflow_p36/bin/dashboard
2019-02-19 00:11:13,037 Starting Bokeh server version 0.12.13 (running on Tornado 5.0.2)
ERROR:__init__() got an unexpected keyword argument 'io_loop'
```

Do I need to downgrade Bokeh? How do I control the port that this is associated with?

**Comment by scttl** [comment]
This actually looks like a bokeh - Tornado compatibility issue. 
 See: https://github.com/bokeh/bokeh/issues/7308 

coach requires bokeh `0.13.0` or higher and it looks like you've got `0.12.13` installed which isn't compatible with Tornado 5. 

If you want finer control over the dashboard serving port, you'll likely need to modify this line it seems: https://github.com/NervanaSystems/coach/blob/master/rl_coach/dashboard.py#L61

**Comment by galleibo-intel** [comment]
Allowing for remote machines with dashboard is fixed with PR #231
